### PR TITLE
Feature/slimlint

### PR DESCRIPTION
**Pull request by mike927**

### CODE REVIEW

**Comment by dropsrobs** [comment]
:+1:

File: lib/guard/slimlint.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    private
+
+    def run(paths = ['.'])
+      result = system "slim-lint #{paths.join(' ')}"
+      if result
+        UI.info 'No Slim offences detected'.green
+      else
+        UI.info 'Slim offences has been detected'.red
+      end
+      check_and_notify(result)
```
Comment (spidzio) [inline lib/guard/slimlint.rb:38]: `notify(result) if notification_allowed?(result)` is fine here.

Comment (mike927) [inline lib/guard/slimlint.rb:38]: I've changed it becuase I can't test this method otherwise.

**Comment by mike927** [comment]
let me know if I can merge

File: README.md [hunk 2]
```diff
@@ -1,15 +1,16 @@
-# Guard::Slim::Lint
+# Guard::SlimLint
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/guard/slim/lint`. To experiment with that code, run `bin/console` for an interactive prompt.
+Guard::SlimLint gem helps developers to automatically runs slim-lint every time a slim file is saved.
```
Comment (spidzio) [inline README.md]: 'automatically run'

**Comment by spidzio** [comment]
:+1:

File: README.md (hunk 2 above)
Comment (mike927) [inline README.md]: you're right

**Pull request merged**